uaclient/clouds/azure.py

```diff
@@ -0,0 +1,55 @@
+import os
+
+from urllib.error import HTTPError
+
+from uaclient.clouds import AutoAttachCloudInstance
+from uaclient import util
+
+
+IMDS_BASE_URL = "http://169.254.169.254/metadata/"
+IMDS_URLS = {
+    "pkcs7": IMDS_BASE_URL + "attested/document?api-version=2019-06-04",
```

Comment:
Where did this API version come from?

Reply:
This came from cmars comment in the spec "We need to use 2019-06-04 or later, so that the subscription ID is in the attested metadata." 
https://docs.google.com/document/d/1PzLLAKpz9x4FlsYFC7iAFN533d_Kl8xMxH9ahAOc5P8/edit#

Reply:
OK, I've asked Casey in doc if he can point us at the origin of that. I'd like to see a comment explaining why we're using that version, as it's obviously _very_ important that we don't change it backwards (which we might be tempted/asked to do as not all Azure partitions have support for this API version: https://docs.microsoft.com/en-us/azure/virtual-machines/windows/instance-metadata-service#service-availability).